code/datums/spells/bloodcrawl.dm

```diff
@@ -149,9 +149,13 @@
 		var/mob/living/simple_animal/demon/slaughter/demon = L
 		demon.devoured++
 		to_chat(victim, "<span class='userdanger'>You feel teeth sink into your flesh, and the--</span>")
+		var/obj/item/organ/internal/regenerative_core/legion/core = victim.get_int_organ(/obj/item/organ/internal/regenerative_core/legion)
+		if(core)
```

Comment:
You will want an istype here for safety, since get_int_organ is potentially unsafe